Model/Game.cs

```diff
@@ -0,0 +1,120 @@
+﻿namespace GameOfLife
+{
+    [Serializable]
+    public class Game
+    {
+        /// <summary>
+        /// Properties.
+        /// </summary>
+        public int Height { get; set; }
+        public int Width { get; set; }
+        public int[,] currentGeneration { get; private set; }
+        public int iterationCount = 1;
+
+        /// <summary>
+        /// Constructor without parameters.
+        /// </summary>
+        public Game()
+        {
+            Height = 16;
+            Width = 16;
+            currentGeneration = new int[Height, Width];
+
+            InitializeBoard();
+        }
+
+        /// <summary>
+        /// Constructor with parameters from user.
+        /// </summary>
+        /// <param name="x Number of rows (height)."></param> 
+        /// <param name="y  Number of columns (width)."></param>
+        public Game(int x, int y)
+        {
+            this.Height = x;
+            this.Width = y;
+
+            currentGeneration = new int[Height, Width];
+
+            InitializeBoard();
+        }
+        /// <summary>
+        /// Initiate current and next generation boards.
+        /// </summary>
+        private void InitializeBoard()
+        {
+            //Loop over cells using range to set live/ dead cells.
+
+            var range = new Random();
+
+            for (int i = 0; i < Height; i++)
```

Comment:
Better to use iteration variables row and column instead of i and j. That will make code readable and there will be no need for comment inside the method.
Comments inside method are acceptable only when logic is intrinsically complex so that it can't be made easier to understand. 
In summary:
change names of iteration variables,
remove comment,
and in addition remove unneeded empty lines.
Use empty line after if, for, switch blocks which is not followed by closing } of enclosing block.
Use empty line before comment inside method and before return statement.

Reply:
Variables i and j renamed to row and column.
Comments removed.
Unnecessary empty lines removed.